### PR TITLE
DOCS-11406-adding limitation about log entries

### PR DESCRIPTION
**Pull request by IsaacEldridge**
For SE-20493

### CODE REVIEW

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -118,6 +118,7 @@ Runtime Fabric on VMs / Bare Metal supports log forwarding to:
 * External log forwarding does not currently join multiline log entries during log file rotation.
 * Runtime Fabric supports only one-way TLS (client to server) verification.
 * You cannot programmatically change the log-patterns for external log forwarders.
+* External log forwarding follows Syslog Protocl RFC5424, and therefore hostnames, application names, and process IDs are not automatically mapped to log entries. These fields are mapped when you xref:forward-logs-to-other-external-services[forward logs to other external services].
```
Comment (sarjeet2013) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: typo: Protocol*

Comment (IsaacEldridge) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: fixed

**Review by kmendiratta-crm — APPROVED** [review]
For 1.9 which uses older version of fluentbit, this is accurate.

**Review by kbostrom — APPROVED** [review]
ce

**Pull request merged**